src/hotspot/share/prims/jvm.cpp

```diff
@@ -2127,10 +2126,14 @@ JVM_ENTRY(jobjectArray, JVM_GetPermittedSubclasses(JNIEnv* env, jclass current))
   Klass* c = java_lang_Class::as_Klass(mirror);
   assert(c->is_instance_klass(), "must be");
   InstanceKlass* ik = InstanceKlass::cast(c);
+  log_trace(class, sealed)("Calling GetPermittedSubclasses for type %s", ik->external_name());
```

Comment:
Consider adding a sealed indication to this log_trace which would allow you to delete line #2179.  Something like:
("Calling GetPermittedSubclasses for %s type %s", ik->external_name(), (ik->is_sealed() ? "sealed" : "non-sealed"))